docs/src/data/frameworks.ts

```diff
@@ -27,9 +33,11 @@ export const REACT_NATIVE_DEPENDENCIES =
   'react-native-safe-area-context amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage react-native-get-random-values react-native-url-polyfill';
 
 export const FRAMEWORK_INSTALL_SCRIPTS = {
-  react: '@aws-amplify/ui-react aws-amplify',
-  vue: '@aws-amplify/ui-vue aws-amplify',
-  angular: '@aws-amplify/ui-angular aws-amplify',
-  flutter: 'amplify_authenticator',
-  'react-native': `@aws-amplify/ui-react-native aws-amplify ${REACT_NATIVE_DEPENDENCIES}`,
+  react: 'npm i @aws-amplify/ui-react aws-amplify',
+  vue: 'npm i @aws-amplify/ui-vue aws-amplify',
+  angular: 'npm i @aws-amplify/ui-angular aws-amplify',
+  flutter: 'flutter pub add amplify_authenticator',
+  android: "implementation 'com.amplifyframework.ui:liveness:1.0.0'",
+  swift: 'add swift install script here',
```

Comment:
When will this be added?

Reply:
This was missed, i'll sync with @atierian

Reply:
Thanks for catching this. 
There isn't an install script for Swift. The package is added through the IDE (Xcode). I don't know enough about the docs structure to say whether `swift:` should be removed or if it should just be an empty string though.

Reply:
Removed the install script for swift for now